app/controllers/discussions_controller.rb

```diff
@@ -19,8 +19,10 @@ class DiscussionsController < ApplicationController
   before_action :stash_comment,    except: %i[ index show add_comment ]
 
   def index
-    # TODO: list of recently-active discussions etc
-    redirect_to root_path
+    count = ( params[ :count ] || 10 ).to_i
+    days  = ( params[ :days  ] || 7  ).to_i
```

Comment:
Consider applying a sensible upper limit for `count` and `days` to mitigate users trying to query excessive amount of data in a request.